staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/schema/cel/compilation.go

```diff
@@ -40,6 +40,14 @@ const (
 	// OldScopedVarName is the variable name assigned to the existing value of the locally scoped data element of a
 	// CEL validation expression.
 	OldScopedVarName = "oldSelf"
+
+	// perCallLimit specify the actual cost limit per CEL validation call
+	// current perCallLimit gives roughly one second for each expression validation call
+	perCallLimit = 20000000
+
+	// RuntimeCELCostBudget is the overall cost budget for runtime CEL validation cost per CustomResource
+	// current RuntimeCELCostBudget gives roughly 10 seconds for CR validation
+	RuntimeCELCostBudget = 200000000
```

Comment:
since these represent dedicated CPU time, both of these are about an order of magnitude higher than I expected... are we really ok with 10 seconds of devoted CPU time per custom resource write validation?

Reply:
This number is fairly high now. I was thinking to start with a higher number and reduce based on performance run. Would you have any suggestions on the CPU time custom resource could consume?

Reply:
at most, I would expect fractions of dedicated CPU per call and less than a second of dedicated CPU for validating the entire resource. I wouldn't start any higher than that, and would actually like to see that ramp even further down as we prove you can do significant amounts of complex validation with much lower limits.

Reply:
I have updated to start with `2000000` for `PerCallLimit` and `20000000` for `RuntimeCELCostBudget`. cc @jpbetz @DangerOnTheRanger